borderland-tool/borderland_tool/lottery/__init__.py

```diff
@@ -1,41 +1,47 @@
 #!/usr/bin/python3
-import csv, random, json
+import csv
+import random
+import json
 
 from datetime import datetime, timezone
 from dateutil import parser
 
+
 class Lottery:
     def __init__(self, pretix, csvfile, quota):
         self.pretix = pretix
         self.csvfile = csvfile
         self.quota = quota
-        self.registered = self.load_csv()
+        self.registrations_csv = self.load_csv()
         self.has_order = self.has_voucher = None
 
     def registrations_to_csv(self):
-        """Retrieve registered users from Pretix plugin and update CSV"""
-        registrations = self.pretix.get_registrations()
-        if self.registered:
-            last = int(self.registered[-1]['id'])
+        """Retrieve registrered users from Pretix plugin and update CSV"""
```

Comment:
is registered according to the dictionary.

Reply:
The word "registrered" does not exist, it was correct as it was; that's why I requested the changes.
But I saw you merged now... could you create a new PR to fix that back? ... I mean is silly, and we are the ones that will only see it ... but it was correct before, so it itches me  a bit ...

Reply:
Aah you mean misspelling in the comment? Sure i can fix that